public void testAssignHospitalException() throws Exception {
    try {
        evilDAO.assignHospital(0L, "");
        fail("DBException should have been thrown");
    } catch (DBException e) {
        assertEquals(EvilDAOFactory.MESSAGE, e.getSQLException().getMessage());
    }
}